cmd/dfget/app/root.go

```diff
@@ -99,7 +99,7 @@ func runDfget() error {
 		util.Printer.Println(fmt.Sprintf("assert context error: %v", err))
 		return err
 	}
-	logrus.Infof("get init config:%v", cfg)
+	logrus.Debugf("get init config:%v", cfg)
```

Comment:
It will output the key information of `dfget` which not just for debugging errors , and I think maybe it's more suitable to use `INFO` level .  WDYT？

Reply:
it will output some logs like this. Print http header is not readable.

2019-08-02 00:10:17.775 INFO sign:2425-1564675817.774 : get init config:{"url":"https://index.docker.io/v2/library/ubuntu/blobs/sha256:7413c47ba209e555018c4be91101d017737f24b0c9d1f65339b97a4da98acb2a","output":"/home/yeya24/.small-dragonfly/dfdaemon/data/c45e5005-7246-4263-8d0b-fc4ce7bd2021","localLimit":20971520,"minRate":65536,"totalLimit":20971520,"callSystem":"com_ops_dragonfly","pattern":"p2p","filter":["Signature","Expires","OSSAccessKeyId"],"header":["X-Forwarded-For:127.0.0.1","User-Agent:docker/18.06.1-ce go/go1.10.3 git-commit/e68fc7a kernel/4.15.0-29deepin-generic os/linux arch/amd64 UpstreamClient(Docker-Client/18.06.1-ce \\(linux\\))","Authorization:Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsIng1YyI6WyJNSUlDK2pDQ0FwK2dBd0lCQWdJQkFEQUtCZ2dxaGtqT1BRUURBakJHTVVRd1FnWURWUVFERXpzeVYwNVpPbFZMUzFJNlJFMUVVanBTU1U5Rk9reEhOa0U2UTFWWVZEcE5SbFZNT2tZelNFVTZOVkF5VlRwTFNqTkdPa05CTmxrNlNrbEVVVEFlRncweE9UQXhNVEl3TURJeU5EVmFGdzB5TURBeE1USXdNREl5TkRWYU1FWXhSREJDQmdOVkJBTVRPMUpMTkZNNlMwRkxVVHBEV0RWRk9rRTJSMVE2VTBwTVR6cFFNbEpMT2tOWlZVUTZTMEpEU0RwWFNVeE1Pa3hUU2xrNldscFFVVHBaVWxsRU1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBcjY2bXkveXpHN21VUzF3eFQ3dFplS2pqRzcvNnBwZFNMY3JCcko5VytwcndzMGtIUDVwUHRkMUpkcFdEWU1OZWdqQXhpUWtRUUNvd25IUnN2ODVUalBUdE5wUkdKVTRkeHJkeXBvWGc4TVhYUEUzL2lRbHhPS2VNU0prNlRKbG5wNGFtWVBHQlhuQXRoQzJtTlR5ak1zdFh2ZmNWN3VFYWpRcnlOVUcyUVdXQ1k1Ujl0a2k5ZG54Z3dCSEF6bG8wTzJCczFmcm5JbmJxaCtic3ZSZ1FxU3BrMWhxYnhSU3AyRlNrL2tBL1gyeUFxZzJQSUJxWFFMaTVQQ3krWERYZElJczV6VG9ZbWJUK0pmbnZaMzRLcG5mSkpNalpIRW4xUVJtQldOZXJZcVdtNVhkQVhUMUJrQU9aditMNFVwSTk3NFZFZ2ppY1JINVdBeWV4b1BFclRRSURBUUFCbzRHeU1JR3ZNQTRHQTFVZER3RUIvd1FFQXdJSGdEQVBCZ05WSFNVRUNEQUdCZ1JWSFNVQU1FUUdBMVVkRGdROUJEdFNTelJUT2t0QlMxRTZRMWcxUlRwQk5rZFVPbE5LVEU4NlVESlNTenBEV1ZWRU9rdENRMGc2VjBsTVREcE1VMHBaT2xwYVVGRTZXVkpaUkRCR0JnTlZIU01FUHpBOWdEc3lWMDVaT2xWTFMxSTZSRTFFVWpwU1NVOUZPa3hITmtFNlExVllWRHBOUmxWTU9rWXpTRVU2TlZBeVZUcExTak5HT2tOQk5sazZTa2xFVVRBS0JnZ3Foa2pPUFFRREFnTkpBREJHQWlFQXFOSXEwMFdZTmM5Z2tDZGdSUzRSWUhtNTRZcDBTa05Rd2lyMm5hSWtGd3dDSVFEMjlYdUl5TmpTa1cvWmpQaFlWWFB6QW9TNFVkRXNvUUhyUVZHMDd1N3ZsUT09Il19.eyJhY2Nlc3MiOlt7InR5cGUiOiJyZXBvc2l0b3J5IiwibmFtZSI6ImxpYnJhcnkvdWJ1bnR1IiwiYWN0aW9ucyI6WyJwdWxsIl19XSwiYXVkIjoicmVnaXN0cnkuZG9ja2VyLmlvIiwiZXhwIjoxNTY0Njc2MTA2LCJpYXQiOjE1NjQ2NzU4MDYsImlzcyI6ImF1dGguZG9ja2VyLmlvIiwianRpIjoibjNENWZUZ1BCQkl2RzBaTzNSVW8iLCJuYmYiOjE1NjQ2NzU1MDYsInN1YiI6ImMyNjgwNDJiLTJkZjItNDNmMi1iNWIwLTU3MjRmNGZhMDBjMSJ9.OsJzEp23RyYlhduDNasf0KpArsMYSX28UeXXWjPibOp1-DOCDgqMNLoquFzk9cs8KVXRGCjsdQhzC8gWkBCaItoKKOK63qsbKMfT_tyuFin3WT9mK9tecWXyJP4wP9GhaQni083fpHWUrme6e1cdnpd5c6aduQlQbmv1EMDKUyS21LbvbqsLAM6_2fT7cgE4B9vvanlGlUYL9rsKp-dasK_H8EI3ky3TUnlJ94JiEK0p4HjaodrjjbQRG7bcwiJ6tv93Jv8mH_khnFqG7SbPv9rzRDHjdOsPJkQh2AZD_KkVPUx5CXHdm4wkDvaUJPZlrenoDAjTpwkYDDOp-sueYw"],"node":["127.0.0.1:8002"],"notbs":true,"dfdaemon":true,"verbose":true,"clientQueueSize":6,"startTime":"2019-08-02T00:10:17.774468667+08:00","sign":"2425-1564675817.774","user":"yeya24","workHome":"/home/yeya24/.small-dragonfly","configFile":["/etc/dragonfly/dfget.yml","/etc/dragonfly.conf"]}

Reply:
Maybe we can define a struct such as `BaseConfig` as the anonymous struct of the `dfget config`. And for the output, we only print the `BaseConfig`. WDYT?